### ISSUE TITLE
EIP/CIP to PLC

### ISSUE THREAD

**Issue by paviano** [gisellevonbingen/Giselle.Net.EtherNetIP#2]
Is possible to use this project to implement a CIP communication with PLC's?

My congrats for this repo, looks good

**Comment by gisellevonbingen** [comment]
Currently, this repo can use only as CIP Master like PLCs.
Not immediately, but I plan to add someday.

What type of PLC are you using?
I'm going to test one of KEYENCE or OMRON PLCs.

**Comment by paviano** [comment]
Yes, it would be amazing I think, in the market there are only low performance/bugged connectors.
I want to integrate the Rockwell PLC. Apperently this package correcly connects to it and read identify information.

Which could be a good starting point? 
Building a new DataProcess with its related CommandItems?

My Ref: 

https://literature.rockwellautomation.com/idc/groups/literature/documents/pm/1756-pm020_-en-p.pdf

**Comment by gisellevonbingen** [comment]
Sorry for late reply.
I realize I have to learn more.

Unfortunately Rockwell PLC is not popularity in my country.
Maybe i can't test connect to plc in real.

But the Ref is looks very nice.
I will try it can be has compatibility.

**Comment by paviano** [comment]
Don't worry, thank you anyway. 
Me neither and these PLC's are very expensive and hard to get, 
I'm currently working on something own by my company.

I let you know if I arrive somewhere, eventually I will open a pull request

**Comment by gisellevonbingen** [comment]
Great. Thank for your interest.
